Utility/integration-test.sh

```diff
@@ -17,10 +20,16 @@ if [ -x "$(command -v flow)" ]; then
 fi
 
 # Move headers in the right place for the Swift PM
-mv Examples/Cocoa/Sources/objc/*.h Examples/Cocoa/Sources/objc/include
+mv Examples/Cocoa/Objc/Sources/objc/*.h Examples/Cocoa/Objc/Sources/objc/include
+
+# Build the Obj-c library
+pushd Examples/Cocoa/Objc
```

Comment:
can we refactor these 4 lines into a simple bash function that takes the path of the package root ? `Examples/Cocoa/Objc` or `Examples/Cocoa/Swift` for example